libnetwork_test.go

```diff
@@ -0,0 +1,349 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	etcdclient "github.com/coreos/etcd/client"
+	"github.com/docker/docker/api/types/network"
+	dockerclient "github.com/docker/docker/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
```

Comment:
nit:  Can we just do a . import like the other test utils?   Makes it read a bit more nicely in the tests.